demo/secret/env/client.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"mosn.io/layotto/sdk/go-sdk/client"
+	runtimev1pb "mosn.io/layotto/spec/proto/runtime/v1"
+)
+
+func main() {
```

Comment:
Can we delete demo/secret/env/client.go , demo/secret/file/client.go and demo/secret/kubernetes/client.go ? 
Since we have demo/secret/common/client.go , we can use one demo to showcase all components

Reply:
we can leave it， simplify some testing